addons/website_sale/static/src/js/website_sale_tour.js

```diff
@@ -0,0 +1,481 @@
+    },
+    {
+        content: "Click for edit address",
+        trigger: 'a:contains("Edit") i',
+    },
+    {
+        content: "Confirm Address",
+        trigger: 'a.btn:contains("Confirm")',
+    },
+    {
+        content: "Select `Wire Transfer` payment method",
+        trigger: '#payment_method label:contains("Wire Transfer")',
+    },
+    {
+        content: "Pay Now",
+        extra_trigger: '#payment_method label:contains("Wire Transfer") input:checked,#payment_method:not(:has("input:radio:visible"))',
+        trigger: 'button[id="o_payment_form_pay"]:visible:not(:disabled)',
+    },
+    {
+        content: "Sign up",
+        trigger: '.oe_cart a:contains("Sign Up")',
+    },
+    {
+        content: "Submit login",
+        trigger: '.oe_signup_form',
+        run: function (){
+            $('.oe_signup_form input[name="password"]').val("1admin@admin");
+            $('.oe_signup_form input[name="confirm_password"]').val("1admin@admin");
+            $('.oe_signup_form').submit();
+        },
+    },
+    {
+        content: "See Quotations",
+        trigger: '.o_portal_docs a:contains("Quotations")',
+    },
+    // Sign in as admin change config auth_signup -> b2b, sale_show_tax -> total and Logout
+    {
+        content: "Open Dropdown for logout",
+        trigger: '#top_menu li.dropdown:visible a:contains("abc")',
+    },
+    {
+        content: "Logout",
+        trigger: '#o_logout:contains("Logout")',
+    },
+    {
+        content: "Sign in as admin",
+        trigger: '#top_menu li a b:contains("Sign in")',
+    },
+    {
+        content: "Submit login",
+        trigger: '.oe_login_form',
+        run: function (){
+            $('.oe_login_form input[name="login"]').val("admin");
+            $('.oe_login_form input[name="password"]').val("admin");
+            $('.oe_login_form input[name="redirect"]').val("/");
+            $('.oe_login_form').submit();
+        },
+    },
+    {
+        content: "Configuration Settings for 'Tax Included' and sign up 'On Invitation'",
+        extra_trigger: '.o_connected_user #wrapwrap',
+        trigger: '#wrapwrap',
+        run: function () {
+            var def1 = rpc.query({
+                model: 'res.config.settings',
+                method: 'create',
+                args: [{
+                    'auth_signup_uninvited': 'b2b',
+                    'show_line_subtotals_tax_selection': 'tax_included',
+                    'group_show_line_subtotals_tax_excluded': false,
+                    'group_show_line_subtotals_tax_included': true,
+                }],
+            });
+            var def2 = def1.then( function (resId) {
+                return rpc.query({
+                    model: 'res.config.settings',
+                    method: 'execute',
+                    args: [[resId]],
+                });
+            });
+            def2.then(function () {
+                window.location.href = '/web/session/logout';
```

Comment:
you can replace with /web/session/logout?redirect=/shop?search=Storage Box so you can remove the two next steps (that clicks on /shop and then do a search)